test-suite-kotlin-client-generator/build.gradle

```diff
@@ -13,9 +11,9 @@ that tests can be ran with Micronaut 4
 
 dependencies {
 
-    kapt("io.micronaut.serde:micronaut-serde-processor")
-    kapt("io.micronaut.validation:micronaut-validation-processor")
-    kapt(mn.micronaut.inject.kotlin)
+    ksp("io.micronaut.serde:micronaut-serde-processor")
```

Comment:
Are you sure you want to switch to KSP? It might be better to have different two projects, one for KSP and one for KAPT.

Reply:
We can create two more test project for it :-) But I already found new problem with KSP. Because of it I can't change KAPT to KSP in `test-suite-kotlin-server-generator`. Right now I haven't time to describe this problem. Will do it later. Wait issue in core ;-)

Reply:
done. Now we have different tests for ksp and kapt